locations/spiders/chick-fil-a.py

```diff
@@ -0,0 +1,25 @@
+from locations.categories import Extras, apply_yes_no
+from locations.storefinders.yext import YextSpider
+from locations.structured_data_spider import clean_instagram, clean_twitter
+
+
+class ChickfilASpider(YextSpider):
+    name = "chick-fil-a"
```

Comment:
I'd vote for chick_fil_a

Reply:
Agree. Please use underscores for the spider name.

Reply:
Done, thanks for the review.

I wasn't sure in this case as the official name has hypens not spaces. Agreed that underscores are best to replace any punctuation or spaces in official names :)